tests/commands/timezone.py

```diff
@@ -196,5 +196,27 @@ def runTest(self):
                                 KickstartDeprecationWarning,
                                 "The option --isUtc will be deprecated in future releases")
 
+
+class F33_TestCase(F32_TestCase):
+    command = "timezone"
+
+    def setUp(self):
+        super().setUp()
+        warnings.simplefilter("error", category=KickstartDeprecationWarning)
```

Comment:
I don't think we should transform warnings into exceptions, because it can break tests for older versions. This is happening to F32 and F33 timezone tests. Please, remove this line from both test cases:

`warnings.simplefilter("error", category=KickstartDeprecationWarning)`

The warnings can be simply tested with:


```
        with self.assertWarns(KickstartDeprecationWarning):
            self.assert_parse("timezone --isUtc Europe/Bratislava")

```

Reply:
Yep, that looks much better than the old code from the F32 test case I have reused. :)